src/Core/Types/PatternType.cs

```diff
@@ -78,8 +79,8 @@ protected override string GetNameInProperFormat(string name, int id)
         {
             StringBuilder sb = new StringBuilder(name);
 
-            sb.Replace("UIA_", "");
-            sb.Replace("Id", "");
+            sb.Replace(DisplayStrings.UIA_, "");
+            sb.Replace(DisplayStrings.Id, "");
```

Comment:
These are used for mapping to a code construct. I would not recommend moving them into a resource file